Dokan/Dokan/Scenes/Login/LoginViewController.swift

```diff
@@ -0,0 +1,90 @@
+//  
+//  LoginViewController.swift
+//  Dokan
+//
+//  Created by Omar Ahmed on 02/07/2022.
+//
+
+import UIKit
+
+class LoginViewController: UIViewController {
+    
+    // MARK: Outlets
+    
+    @IBOutlet weak var emailTextField: UITextField!
+    @IBOutlet weak var passwordTextField: UITextField!
+    @IBOutlet weak var signInBtn: UIButton!
+    @IBOutlet weak var forgotPasswordBtn: UIButton!
+    @IBOutlet weak var signUpBtn: UIButton!
```

Comment:
```suggestion
    @IBOutlet private(set) weak var emailTextField: UITextField!
    @IBOutlet private(set) weak var passwordTextField: UITextField!
    @IBOutlet private(set) weak var signInButton: UIButton!
    @IBOutlet private(set) weak var forgotPasswordButton: UIButton!
    @IBOutlet private(set) weak var signUpButton: UIButton!
```

- Take care of access modifiers
- Please don't user btn, lbl 😁